static/src/javascripts/projects/common/modules/commercial/survey/survey-simple.js

```diff
@@ -0,0 +1,75 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/template',
+    'common/modules/user-prefs',
+    'common/views/svgs',
+    'text!common/views/commercial/survey/survey-simple.html',
+    'lodash/arrays/uniq'
+], function (
+    bean,
+    fastdom,
+    $,
+    template,
+    userPrefs,
+    svgs,
+    surveySimpleTemplate,
+    uniq
+) {
+    var surveySimple = function (config) {
+        this.config = config || {};
+        this.id = this.config.id;
+        this.prefs = 'overlay-messages';
+        this.closePermanently = this.config.closePermanently || false;
+        this.bannerTmpl = template(surveySimpleTemplate,
+            {
+                surveyHeader: this.config.surveyHeader,
+                surveyText: this.config.surveyText,
+                buttonText: this.config.buttonText,
+                buttonLink: this.config.buttonLink,
+                buttonDataLink: this.config.membershipDataLink,
+                showCloseBtn: this.config.showCloseBtn,
+                arrowWhiteRight: svgs('arrowWhiteRight'),
+                marque36icon: svgs('marque36icon'),
+                crossIcon: svgs('crossIcon'),
+                surveyOverlaySimple: svgs('surveyOverlaySimple')
+            });
+    };
+
+    surveySimple.prototype.attach = function () {
+        if (!this.hasSeen()) {
+            fastdom.write(function () {
+                $(document.body).append(this.bannerTmpl);
+
+                if (this.config.showCloseBtn) {
+                    bean.on(document, 'click', $('.js-survey-close'), function () {
+                        $('.js-survey-overlay').addClass('u-h');
+                        if (this.closePermanently) {
+                            this.remember();
+                        }
+                    }.bind(this));
+                }
+            }.bind(this));
+        }
+    };
+
+    surveySimple.prototype.hasSeen = function () {
+        var messageStates = userPrefs.get(this.prefs);
+        return messageStates && messageStates.indexOf(this.id) > -1;
+    };
+
+    surveySimple.prototype.remember = function () {
```

Comment:
I think this method name is maybe a little broad? I personally would call the method `closePermanently` and the config item something else (like `shouldClosePermanently`, maybe)